app/src/main/java/org/eyeseetea/malariacare/data/database/utils/populatedb/PopulateDB.java

```diff
@@ -975,4 +978,12 @@ public static void wipeOrgUnitsAndEvents() {
                 SurveyScheduleDB.class,
                 UserDB.class).toArray());
     }
+
+    public static void wipeMedia() {
```

Comment:
if we save download media files under specific media folder, when we want to remove media files is more simple and is not necessary get media paths previously